NEWS.md

```diff
@@ -110,43 +144,6 @@ New architectures
   * [Power](https://github.com/JuliaLang/julia/issues?utf8=%E2%9C%93&q=label%3Apower):
     [#16455], [#16404]
 
-Breaking changes
```

Comment:
I think breaking changes are more important than compiler and library improvements for people who will be reading this file. deprecations are important to mention too (and way incomplete here) but can maybe be a bit later.